public/orbs-public-beta-own-infra.md

```diff
@@ -0,0 +1,74 @@
+# Running Orbs v2 on your own infrastructure
+
+## Prerequisites
+
+* A Linux machine with Docker Swarm
+* Boyar v1.1.1 [binary download](https://s3.amazonaws.com/orbs-network-releases/infrastructure/boyar/boyar-v1.1.1.bin)
+* Connection to Ethereum node (`$ETHEREUM_ENDPOINT` url)
+* Directories `/var/efs`, `/var/efs/boyar-status` and `/var/efs/boyar-logs` should be created on the machine and have enough space (separate filesystem is recommended, you can use NFS if you like)
+
+## JSON configuration files
+
+### `mgmt.json`
+
+You need to replace `$ETHEREUM_ENDPOINT` with a URL of a real Ethereum node (you can use Infura free tier or your own node).
+
+```json
+{
+  "orchestrator": {
+    "DynamicManagementConfig": {
+      "Url": "http://localhost:7666/node/management",
+      "ReadInterval": "1m",
+      "ResetTimeout": "30m"
+    },
+    "storage-driver": "local",
+    "storage-mount-type": "bind"
+  },
+  "services": {
+    "management-service": {
+      "InternalPort": 8080,
+      "ExternalPort": 7666,
+      "DockerConfig": {
+        "Image": "orbsnetwork/management-service",
+        "Tag": "v1.1.1",
```

Comment:
we changed this to be "bootstrap". 1.1.1 is no longer the correct version and we did not want to make the instructions sensitive to upgrades